waspc/cli/src/Wasp/Cli/Command/Build.hs

```diff
@@ -56,5 +59,12 @@ buildIO waspProjectDir buildDir = compileIOWithOptions options waspProjectDir bu
       CompileOptions
         { externalCodeDirPath = waspProjectDir </> Common.extCodeDirInWaspProjectDir,
           isBuild = True,
-          sendMessage = cliSendMessage
+          sendMessage = cliSendMessage,
+          -- Ignore "DB needs migration warnings" during build, as that is not a required step.
+          warningsFilter =
+            filter
+              ( \case
+                  GeneratorNeedsMigrationWarning _ -> False
+                  _ -> True
+              )
```

Comment:
Any cleaner way to do this? If I _only_ wanted `GeneratorNeedsMigrationWarning` I could use List comprehensions (a la `[x | Just x <- xs]`), but unsure of a similar way of doing that if I want everything _except_ a particular Data Constructor.

Reply:
Ok, thanks for the sanity check, I appreciate it.